roles/set_custom_fact/files/device_list.fact

```diff
@@ -2960,10 +2960,13 @@
 			}
 		},
 		"kb-bsf-scaleway": {
-			"kalite": {
-				"activated": "True",
-				"version": "0.16.9",
-				"language": ["fr"]
+			"kolibri": {
+				"activated": "True"
```

Comment:
Language, not yet. Soon. :)
For now, not much can be configured, appart from plugins. Which will come soon enough.